test/test.js

```diff
@@ -19,4 +20,130 @@ describe('Grammar', function() {
       assert.equal(output, false);
     });
   });
+
+
+  describe("#mandatory_markers",function(){
+	it('the 4 mandatory_markers are id,c,p,v ',function(){
+  		fs.readFile('test/test_simple_ID_C_P_V_correct.usfm', function(err, data) {
```

Comment:
As much is possible, we'll use minimal string examples right within the code to allow for easy reference and readability even if the examples are completely synthetic.